base-images/python3.8-ubuntu20.04.Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && \
     bzip2 ca-certificates libglib2.0-0 libsm6 libxext6 libxrender1 mercurial \
     procps subversion wget \
     # envd dependencies
-    python3 curl openssh-client git tini sudo python3-pip zsh vim \
+    curl openssh-client git tini sudo python3-pip zsh vim \
```

Comment:
Also remove the python3-pip here please

Reply:
I split it in two commit. sry

Reply:
LGTM!

Reply:
Should I also rename the Dockerfile? maybe python-ubuntu20.04? But I think I need help to fix other hardcode such as image base in the code and CI process.